config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,24 @@
+if Rails.env.production?
+  class Rack::Attack
+    Rack::Attack.cache.store = ActiveSupport::Cache::RedisCacheStore.new(url: ENV['REDIS_URL'])
+
+    API_KEY_LIMITS = JSON.parse(ENV['GREEN_LANES_API_KEYS'])
```

Comment:
I'm a little lost on how this controls just for /green_lanes/ scoped api calls?

Reply:
This is a good point, I added logic to only throttle `/green_lanes/goods_nomenclatures` endpoint. Also removed the default throttle method as it doesn't make sense. If the api-key does not exist in the json, the the request will be blocked anyway.